lib/experiments-page.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Bootstraping the Gutenberg experiments page.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * The main entry point for the Gutenberg experiments page.
+ *
+ * @since 5.2.3
+ *
+ * @param string $page The page name the function is being called for, `'gutenberg_customizer'` for the Customizer.
+ */
+function the_gutenberg_experiments( $page = 'gutenberg_page_gutenberg-experiments' ) {
+	?>
+	<div
+		id="experiments-editor"
+		class="blocks-experiments-container"
+	>
+	<?php settings_errors(); ?>
+	<form method="post" action="options.php">
+		<?php settings_fields( 'gutenberg-experiments' ); ?>
+		<?php do_settings_sections( 'gutenberg-experiments' ); ?>           
+		<?php submit_button(); ?>
+	</form>
+	</div>
+	<?php
+}
+
+/**
+ * Set up the experiments settings.
+ *
+ * @since 5.2.3
+ */
+function gutenberg_initialize_experiments_settings() {
+	add_settings_section(
+		'gutenberg_experiments_section',
+		__( 'Experiment settings', 'gutenberg' ),
+		'gutenberg_display_experiment_section',
+		'gutenberg-experiments'
+	);
+	add_settings_field(
+		'gutenberg-widgets-screen',
+		__( 'Widgets Screen', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Widgets Screen', 'gutenberg' ),
+			'id'    => 'gutenberg-widgets-screen',
+		)
+	);
+	add_settings_field(
+		'gutenberg-legacy-widget-block',
+		__( 'Legacy Widget Block', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Legacy Widget Block', 'gutenberg' ),
```

Comment:
That's not really accessible I think. It should be more like "Enable legacy widget block"